app/views/suggestions/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<header>
+
+  <ul class="nav">
+    <% if current_user %>
+      <li><%= link_to "Your Favorites", your_favorites_path(session[:user_id]), method: :get %></li>
```

Comment:
`GET` is the default method for `link_to`, so you don't have to specify it here :)